source/index.html

```diff
@@ -1,12 +1,139 @@
 <!DOCTYPE html>
-<html lang="ru">
+<html class="page" lang="ru">
   <head>
     <meta charset="utf-8">
-    <title>HTML Academy: Седона</title>
+    <title>Седона</title>
   </head>
-  <body>
+  <body class="page__body">
+    <header class="main-header">
+      <a class="main-header__logo" href="#">
+        <img src="#" width="185" height="96" alt="Седона">
+      </a>
+      <nav class="main-nav">
+        <button class="main-nav__toggle" type="button">
+          <span class="visually-hidden">Открыть меню</span>
+        </button>
+        <div class="main-nav__wrapper">
+          <ul class="main-nav__list site-list">
+            <li class="site-list__item site-list__item--active">
+              <a class="site-list__link" href="#">Главная</a>
+            </li>
+            <li class="site-list__item">
+              <a class="site-list__link" href="catalog.html">Фото и видео</a>
+            </li>
+            <li class="site-list__item">
+              <a class="site-list__link" href="form.html">Форма отзыва</a>
+            </li>
+            <li class="site-list__item">
+              <a class="site-list__link" href="https://htmlacademy.ru/intensive/adaptive">HTML Academy</a>
+            </li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+    <main class="main-container">
+      <section class="intro">
+        <h1 class="visually-hidden">Седона - небольшой городок в Аризоне</h1>
+        <div class="intro__wrapper">
+          <img class="intro__slogan" src="#" width="282" height="214" alt="Добро пожаловать в Седону. Здесь лучше, чем в Гранд-Каньоне">
+          <div class="intro__text-block text-block">
+            <div class="text-block__title">Седона— небольшой городок в Аризоне, заслуживающий большего!</div>
+            <div class="text-block__description">Рассмотрим 5 причин, по которым Седона круче, чем гранд каньон!</div>
+          </div>
+        </div>
+      </section>
 
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/adaptive">HTML и CSS. Адаптивная вёрстка и автоматизация</a>».</p>
+      <section class="advantages">
+        <h2 class="visually-hidden">Преимущества Седоны</h2>
+        <ul class="advantages__list">
+          <li class="advantages__item advantages__item--primary">
+            <h3 class="advantages__title">Настоящий городок</h3>
```

Comment:
Тут можно поиграться с примесями. Вот как пример
```suggestion
        <ul class="advantages__list advantages-list">
          <li class="advantages-list__item advantages-list__item--primary advantage">
            <h3 class="advantage__title">Настоящий городок</h3>
```